Makefile

```diff
@@ -1,8 +1,9 @@
 OS=$(shell uname)
 VERSION=v1.12.0
 CGO_LDFLAGS="-Lgen/third_party/rplidar_sdk-release-${VERSION}/sdk/output/${OS}/Release/"
+BUILD_CHANNEL?=local
```

Comment:
[opt] You could probably remove the BUILD_CHANNEL and hardcode "latest" wherever BUILD_CHANNEL is used. Or you could leave this in place, in case we ever want to create a stable release channel.